changelog/199.docs.md

```diff
@@ -0,0 +1,2 @@
+- Added database views for the CMIP7 mip era (previous views were CMIP6Plus only).
+- Fixed up the source ID for SOLARIS-HEPPA in the delivery summary view.
```

Comment:
Also corrected the mip_era entry (CMIP6Plus -> CMIP7) in the SOLARIS-HEPPA-CMIP-4-6 source_id registration

Reply:
Nice, updated